<launch>
 
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find social_sim_ros)/config/logging.conf"/>

  <node pkg="move_base" type="move_base" respawn="true" name="move_base" output="screen">

    <rosparam file="$(find social_sim_ros)/params/kuri/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find social_sim_ros)/params/kuri/costmap_common_params.yaml" command="load" ns="local_costmap" />
    
    <rosparam file="$(find social_sim_ros)/params/kuri/map_nav_params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find social_sim_ros)/params/kuri/map_nav_params/global_costmap_params.yaml" command="load" />
    
    <rosparam file="$(find social_sim_ros)/params/kuri/base_local_planner_params.yaml" command="load" />
    <rosparam file="$(find social_sim_ros)/params/kuri/move_base_params.yaml" command="load" />
    
    <remap from="cmd_vel" to="mobile_base_controller/cmd_vel" />

  </node>

  <node pkg="social_sim_ros" type="pose_to_people" name="pose_to_people" output="screen"/>

</launch>
